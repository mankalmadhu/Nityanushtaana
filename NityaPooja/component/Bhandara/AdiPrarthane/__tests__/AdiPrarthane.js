import 'react-native';
import React from 'react';
import ಆದಿಪ್ರಾರ್ಥನೆ from '../AdiPrarthane';

import Renderer from 'react-test-renderer';

jest.mock('../../../Avarana/Avarana',()=>'ಆವರಣ');
jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');
jest.mock('../../../Vishaya/Viniyoga',()=>'ವಿನಿಯೋಗ');

test("ಆದಿಪ್ರಾರ್ಥನೆ" , () => {

expect(Renderer.create(
    <ಆದಿಪ್ರಾರ್ಥನೆ />
).toJSON()).toMatchSnapshot();

});