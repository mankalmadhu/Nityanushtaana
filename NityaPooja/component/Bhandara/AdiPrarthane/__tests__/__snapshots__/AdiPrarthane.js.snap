// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಆದಿಪ್ರಾರ್ಥನೆ 1`] = `
<ಆವರಣ>
  <ಶೀರ್ಷಿಕೆ>
    ಪ್ರಾರ್ಥನೆ
  </ಶೀರ್ಷಿಕೆ>
  <ವಿನಿಯೋಗ>
    ಶುಕ್ಲಾ೦ಬರಧರ೦ ವಿಷ್ಣು೦ ಶಶಿ ವರ್ಣ೦ ಚತುರ್ಭುಜ೦ | ಪ್ರಸನ್ನ ವದನ೦ ಧ್ಯಾಯೇತ್ ಸರ್ವ ವಿಘ್ನೋಪ ಶಾ೦ತಯೇ ||
  </ವಿನಿಯೋಗ>
</ಆವರಣ>
`;
